### PR TITLE
Enginko

### PR DESCRIPTION
**Pull request by niccoloenginko**
#### Summary

...

#### Changes

- Added the decoder files for the new Level sensors
- Added the yaml files for the new EGK sensors (EGK-LW20L00, EGK-LW20W00, EGK-LW22PLG)
- Updated various values in some of the sensors

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/index.yaml [hunk 1]
```diff
@@ -430,11 +430,11 @@ vendors:
     ouis:
       - 0002CC
 
-  - id: mcf88
```
Comment (Jaime-Trinidad) [inline vendor/index.yaml:433]: Hi Franco, do not delete mcf88 this can affect users with this devices, add as a new vendor engiko

**Review by Jaime-Trinidad — APPROVED** [review]

**Comment by francozampicinini** [comment]
Hi, any update? Thanks.

**Comment by Jaime-Trinidad** [comment]
Hi @niccoloenginko when I validate there is an error with a deleted image
 `mcf88: Error: stat ./vendor/mcf88/enginko_logo.svg: ENOENT`
Can you review it, it comes from the first changes.

**Comment by francozampicinini** [comment]
Hi, my colleague is out of office today.

 

Please can you copy the enginko_logo.svg https://github.com/TheThingsNetwork/lorawan-devices/tree/master/vendor/enginko

 

To https://github.com/TheThingsNetwork/lorawan-devices/tree/master/vendor/mcf88 ?

 

Thanks in advance, 

Franco Zampicinini

 

 

Da: Jaime Trinidad ***@***.***> 
Inviato: giovedì 4 agosto 2022 14:34
A: TheThingsNetwork/lorawan-devices ***@***.***>
Cc: francozampicinini ***@***.***>; Comment ***@***.***>
Oggetto: Re: [TheThingsNetwork/lorawan-devices] Enginko (PR #491)

 

Hi @niccoloenginko <https://github.com/niccoloenginko>  when I validate there is an error with a deleted image
mcf88: Error: stat ./vendor/mcf88/enginko_logo.svg: ENOENT
Can you review it, it comes from the first changes.

**Review by Jaime-Trinidad — APPROVED** [review]

**Comment by Jaime-Trinidad** [comment]
Hi @niccoloenginko there are some conflicts to resolve with the files can you check them so I can run the validate check

**Review by Jaime-Trinidad — APPROVED** [review]

**Comment by Jaime-Trinidad** [comment]
Hi @niccoloenginko @francozampicinini any update here?

**Review by Jaime-Trinidad — APPROVED** [review]

**Pull request merged**